standard/clause_7_core.adoc

```diff
@@ -171,25 +207,328 @@ Access paths follow the XML Schema of http://docs.opengeospatial.org/is/09-146r6
 
 To access coverage service constituents, such as formats supported, OGC 14-121 Web Query Service provides guidance, see https://github.com/opengeospatial/ogc_api_coverages/blob/master/CIS%2BWCS-standards/14-121_Web-Query-Service_2016-06-19.pdf[https://github.com/opengeospatial/ogc_api_coverages/blob/master/CIS%2BWCS-standards/14-121_Web-Query-Service_2016-06-19.pdf].
 
+[[collections]]
+==== Collections
+
+TBD
+
+.Dependencies
+[width="90%"]
+|====
+|http://www.opengis.net/spec/ogcapi_common/1.0/req/collections
+|====
+
+===== Operation
+
+This operation is defined in the `Collections` conformance class of API-Common. No modifications are needed to support `Coverage` resources. The `Collections` conformance class describes two ways of performing this operation:
+
+. Follow the `/collections` path
+. Follow the `service-desc` link on the landing page
+
+Support for both the `/collections` path and the `dataset` link is required by API-Common.
+
+===== Response
+
+[#collections-schema,reftext='Collections Response Schema']
+.Collections Response Schema
+[source,yaml]
+----
+include::openapi/schemas/Common/collections.yaml[]
+----
+
+The following JSON fragment is an example of a response to an OGC API-Coverages Collections operation.
+
+[#conllections-example,reftext=`Collections Example`]
+.Collections Example
+[source,json]
+----
+include::examples/JSON/collections_example.json[]
+----
+
+===== Error situations
+
+See <<http_status_codes>> for general guidance.
+
+[[collectionInfo]]
+==== Collection Information
+
+TBD
+
+.Dependencies
+[width="90%"]
+|====
+|http://www.opengis.net/spec/ogcapi_common/1.0/req/collections
+|====
+
+===== Operation
+
+This operation is defined in the `Collections` conformance class of API-Common. No modifications are needed to support `Coverage` resources. The `Collections` conformance class describes two ways of performing this operation:
+
+. Follow the `/collections/{collectionId}` path
+. Follow the `service-desc` link on the landing page
```

Comment:
Really `service-desc`?

Reply:
Also shouldn't that be a link from a collections page rather than the landing page?

Reply:
Telco 20191218: Need to add `coverage` along `items` as link relation in Common and use here.

Reply:
Telco 20191218: Need to check with IANA defined relations.